site/en/blog/chrome-58-deprecations/index.md

```diff
@@ -0,0 +1,307 @@
+separate path. The design had many side-effects including mouse interactions
+firing `TouchEvents`, all mouse buttons appearing as *left* mouse buttons, and
+`MouseEvents` being suppressed by `TouchEvents`.
+
+Starting with Chrome 58, a mouse on Android M or later will:
+
+* No longer fire `TouchEvents`.
+* Fire a consistent sequence of `MouseEvents` with appropriate buttons and
+  other properties.
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/cNaFvMaYtNA/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5642080642662400) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=468806)
+
+## Remove case-insensitive matching for usemap attribute
+
+The `usemap` attribute was formerly defined as caseless. Unfortunately
+implementing this was complicated enough that no browsers implemented it
+correctly. Research suggested that such a complicated algorithm is unnecessary,
+and even ASCII case-insensitive matching is unnecessary.
+
+Consequently, the specification was updated so that case-sensitive matching is
+applied. The old behavior was deprecated in Chrome 57, and is now removed.
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/8pHdFzN0YQc/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5760965337415680) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=659464)
+
+## Remove content-initiated top frame navigations to data URLs
+
+Because of their unfamiliarity to non-technical browser users, we're
+increasingly seeing the `data:` scheme being used in spoofing and phishing
+attacks. To prevent this, we're blocking web pages from loading `data:` URLs
+in the top frame. This applies to `&lt;a&gt;` tags, `window.open`,
+`window.location` and similar mechanisms. The `data:` scheme will still work for
+resources loaded below by a page.
+
+This feature will be removed in Chrome 60.
+
+[Intent to Remove](https://groups.google.com/a/chromium.org/d/topic/blink-dev/GbVcuwg_QjM/discussion) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5669602927312896) &#124;
+[Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=684011&desc=2)
+
+## Remove deprecated names for motion path properties
+
+Motion path CSS properties allow authors to animate any graphical object along
+an author-specified path. In compliance with the spec, several properties were
+[implemented in Chrome 45](https://www.chromestatus.com/feature/6190642178818048).
+The names of these properties were changed in the spec in mid 2016. Chrome
+implemented the
+[new names in Chrome 55 and Chrome 56](https://www.chromestatus.com/feature/6390764217040896).
+Console deprecation warnings were also implemented.
+
+In Chrome 58, the old property names are being removed. The affected properties
+and their new names are shown below.
+
+| Removed Property | Current Name |
```

Comment:
This will need to be a table in HTML as per the handbook https://developer.chrome.com/docs/handbook/components/#tables